app.vue

```diff
@@ -1,88 +1,20 @@
-<template>
-    <div class="flex flex-col justify-between min-h-screen">
-        <SectionTheMenu />
-        <NuxtPage class="mb-auto pt-24 pb-4" />
-        <SectionTheFooter />
-        <BannerAnnouncements
-            v-if="showBanner"
-            :name="bannerName"
-            message=""
-            short-msg=""
-            link=""
-            button-text="Click Here"
-        />
-    </div>
-</template>
-
-<script>
-export default {
-    data: function () {
-        return {
-            showBanner: false,
-            bannerName: "announcement-banner",
-        };
-    },
-    mounted: function () {
-        // uncomment to show banner on load
-        // this.showBanner =
-        //   window.sessionStorage.getItem(this.bannerName) !== "false";
-    },
-};
+<script setup lang="ts">
+useServerSeoMeta({
+  title: 'Dev Edmonton Society',
+  titleTemplate: '%s | Dev Edmonton Society',
+  description: 'Dev Edmonton Society is a community of developers, designers, and tech enthusiasts in Edmonton, Alberta.',
```

Comment:
is the SEO description supposed to be shorter? Like 140-160 characters?

I assume this would be too log:

Fostering a stronger software development community in Edmonton. We are made up of people who believe that an impactful, diverse, and collaborative developer community enriches us all. We realize this vision by sharing knowledge and ideas, understanding the community and its needs, promoting events and initiatives, and supporting technology meetups and organizations.

Reply:
yes actually, the ogImage look like this: https://devedmonton.vercel.app/__og-image__/image/og.png

it would be nicer if we could shorten the description a bit :)

Reply:
this one is the fallback description, if the page doesn't have a desc. this would show up